### PR TITLE
Update README (firmware deployment instructions, updates, and corrections)

### PR DESCRIPTION
**Pull request by trisapeace**

### CODE REVIEW

File: server/README.md [hunk 1]
```diff
@@ -56,55 +56,157 @@ then create it from the correct `.env` file (for example, on the Sensors Admin s
...
 
 We use [helm](https://helm.sh/docs/) to manage our deployments. Helm allows us to deploy different instances of the application corresponding to dev, staging, and production environments as different 'releases'. Before updating production, we run smoke tests to verify that there's nothing obviously wrong with a release candidate.
 
-### Deploy and run the smoke tests on Staging
-
 1. Run `ssh brave@sensors-admin.brave.coop`
 
 1. cd into the `~/BraveSensor/server` directory and run `git checkout production && git pull origin production` to get the latest version of the helm chart
 
-1. Run the command `helm upgrade staging --set secretName=sensor-staging --set image.tag=<your new container tag> ~/BraveSensor/server/sensor-helm-chart` to deploy the latest version to staging
```
Comment (sampathsatti) [inline server/README.md:69]: @trisapeace is there a reason we don't need to set the image tag when we deploy to staging?

File: server/README.md [hunk 2]
```diff
@@ -56,55 +56,157 @@ then create it from the correct `.env` file (for example, on the Sensors Admin s
...
+
+1. on your local machine, open Visual Studio Code in the `firmware/boron-ins-fsm` directory and open the Particle Workbench extension
+
+1. run `git checkout production && git pull origin production` to get the latest version of the the code
+
+1. click Target --> "Configure for device" and type `2.0.0` to choose the OS and then `boron` to choose the device
+
+1. click Compile --> "Local Compile"
+
+1. copy the generated `/target/2.0.0/boron/boron-ins-fsm.bin` file somewhere to keep for reference. Rename to `v<version number>\_dev.bin`
```
Comment (sampathsatti) [inline server/README.md:83]: Providing an example of the bin file can be renamed to might be useful, for example `v50001_dev.bin`

File: server/README.md [hunk 3]
```diff
@@ -56,55 +56,157 @@ then create it from the correct `.env` file (for example, on the Sensors Admin s
...
+
+1. in your browser navigate to Particle Console --> Sandox --> BetaTest Borons --> Firmware
```
Comment (sampathsatti) [inline server/README.md:85]: Typo `Sandox->Sandbox`

File: server/README.md [hunk 4]
```diff
@@ -56,55 +56,157 @@ then create it from the correct `.env` file (for example, on the Sensors Admin s
...
+1. run `git checkout production && git pull origin production` to get the latest version of the the code
 
-   1. Send a notification to all the responder phones informing them that everything is back online
+1. in `BraveSensorProductionFirmware.ino`, change `BRAVE_PRODUCT_ID` to the ID of the "Production Sensor Devices" project (`15479`) [NOTE: do NOT commit this change]
+
+1. click Target --> "Configure for device" and type `2.0.0` to choose the OS and then `boron` to choose the device
+
+1. click Compile --> "Local Compile"
+
+1. copy the generated `/target/2.0.0/boron/boron-ins-fsm.bin` file somewhere to keep for reference. Rename to `v<version number>\_production.bin`
```
Comment (sampathsatti) [inline server/README.md:147]: Again, providing an example of the renamed bin file might be useful, for example `v50001_production.bin`

Comment (trisapeace) [reply server/README.md]: Done!

File: server/README.md (hunk 3 above)
Comment (trisapeace) [reply server/README.md]: Done.

File: server/README.md (hunk 2 above)
Comment (trisapeace) [reply server/README.md]: Done.

File: server/README.md (hunk 1 above)
Comment (sampathsatti) [reply server/README.md:69]: That makes sense Theresa, I agree with not copy pasting another image tag, if it takes it from `values.yaml` by default.

**Pull request merged**